backend/src/backend/db/models.py

```diff
@@ -71,37 +71,37 @@ class Country(Base):
 class Mirror(Base):
     __tablename__ = "mirror"
 
-    base_url: Mapped[str] = mapped_column(primary_key=True)
+    id: Mapped[str] = mapped_column(primary_key=True)  # hostname of a mirror URL
+    base_url: Mapped[str]
     enabled: Mapped[bool]
     # metadata of a mirror from MirroBrain (https://mirrorbrain-docs.readthedocs.io/en/latest/mirrors.html#displaying-details-about-a-mirror)
-    id: Mapped[Optional[str]]
-    region: Mapped[Optional[str]]
-    asn: Mapped[Optional[str]]
-    score: Mapped[Optional[int]]
-    latitude: Mapped[Optional[float]]
-    longitude: Mapped[Optional[float]]
-    country_only: Mapped[Optional[bool]]
-    region_only: Mapped[Optional[bool]]
-    as_only: Mapped[Optional[bool]]
-    other_countries: Mapped[Optional[List[str]]]
+    region: Mapped[str | None]
+    asn: Mapped[str | None]
+    score: Mapped[int | None]
+    latitude: Mapped[float | None]
+    longitude: Mapped[float | None]
+    country_only: Mapped[bool | None]
+    region_only: Mapped[bool | None]
+    as_only: Mapped[bool | None]
+    other_countries: Mapped[list[str] | None]
 
     country_code: Mapped[str] = mapped_column(
         ForeignKey("country.code"),
         init=False,
     )
-    country: Mapped["Country"] = relationship(back_populates="mirrors", init=False)
+    country: Mapped[Country] = relationship(back_populates="mirrors", init=False)
 
 
 class Worker(Base):
     __tablename__ = "worker"
     id: Mapped[UUID] = mapped_column(
         init=False, primary_key=True, server_default=text("uuid_generate_v4()")
     )
+    # RSA public key for generating access tokens needed to make request to
+    # the web server
```

Comment:
We want the column to be named accordingly: `pubkey_pkcs8` for instance. I also said we should store the fingerprint so `pubkey_fingerprint` as well